pkg/node/provisioners/utilwrappers/fs_operations.go

```diff
@@ -98,6 +98,35 @@ func (fsOp *FSOperationsImpl) PrepareAndPerformMount(src, dst string, bindMount,
 		if wasCreated {
 			_ = fsOp.RmDir(dst)
 		}
+
+		if srcInfo, err := os.Stat(src); err != nil {
+			ll.Debugf("failed to get src stat: %s", err)
+			if os.IsNotExist(err) {
+				ll.Debugf("src path (%s) is not exists", src)
+			}
+		} else {
+			ll.Debugf("Stat of src with failed mount: %s", srcInfo)
+		}
+
+		isSrcMounted, err := fsOp.IsMounted(src)
+		if err != nil {
+			ll.Debugf("failed to execute isMount: %s", err)
+		}
+		if !isSrcMounted {
+			ll.Debugf("Src is not mounted")
+		} else {
+			if srcMount, err := fsOp.FindMountPoint(src); err != nil {
+				ll.Debugf("failed to find mountPoint for src: %s", err)
```

Comment:
Are those `failed` log messages errors? Do we need to print warnings at least?

Reply:
I thought, all this block would print optional debug info. But with warnings it looks better, it's not possible to fail these operations in normal state. Fixed